src/constants/properties.constant.ts

```diff
@@ -54,4 +54,23 @@ export const PROPERTIES = {
       limitRevolvingIndicator: true,
     },
   },
+  DEAL_INVESTOR: {
+    DEFAULT: {
+      sectionIdentifier: '00',
+      lenderType: {
+        lenderTypeCode: '500',
+      },
+      involvedParty: {
+        partyIdentifier: '00000000',
+      },
+      dealStatus: {
+        dealStatusCode: 'A',
+      },
+      customerAdvisedIndicator: true,
+      userDefinedCode1: 'N',
+      limitRevolvingIndicator: true,
+      expirationDate: '2050-12-31T00:00:00Z',
+      contractPercentage: 100, // TO-DO: check
```

Comment:
Is this always 100? It was hardcoded in MuleSoft, but there was a TO-DO to get it from the request instead? If it's always 100 I'll make it a constant.

Reply:
Always 100 %, I think having it in properties is fine, but could be separate 100% constant

Reply:
I think I'll just leave it in properties then